app/experimenter/nimbus-ui/src/lib/visualization/constants.ts

```diff
@@ -219,7 +220,7 @@ export const CONVERSION_METRIC_COLUMNS = [
 // display in the count metric table from left to right.
 export const COUNT_METRIC_COLUMNS = [
   {
-    name: "Count",
+    name: "Absolute Count",
```

Comment:
I added the word "Absolute" for clarity. These metric tables show absolute and relative values - "Relative Improvement" is already stated above the confidence intervals.

Reply:
Yes this makes all the difference in the world, helps me understand the page a lot better thank you 🙏